Formula/bazelisk.rb

```diff
@@ -1,54 +0,0 @@
-# Copyright 2019 The Bazel Authors. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-class Bazelisk < Formula
-  desc "User-friendly launcher for Bazel"
-  homepage "https://github.com/bazelbuild/bazelisk"
-  url "https://github.com/bazelbuild/bazelisk/releases/download/v1.4.0/bazelisk-darwin-amd64"
-  version "1.4.0"
-
-  # To generate run:
-  # curl -L -N -s https://github.com/bazelbuild/bazelisk/releases/download/v1.4.0/bazelisk-darwin-amd64 | shasum -a 256
-  # on macOS
-  sha256 "2f68e450230e95a089018833eb27f217bca8ba145e38a850129b3ac82ccb1052"
-
-  bottle :unneeded
-
-  conflicts_with "bazelbuild/tap/bazel", :because => "Bazelisk replaces the bazel binary"
-
-  resource "bazel_zsh_completion" do
```

Comment:
Can we migrate the bazel_zsh_completion over to Homebrew's bazelisk formula first? It was recently contributed in https://github.com/bazelbuild/homebrew-tap/pull/89.

Reply:
yeah, of course, I can include those changes in the core side.

Reply:
How does that file got generated? Any reference?

Reply:
Nice, thank you!

I had a look and I think that's actually manually written and maintained code (as opposed to the bash completion, which seems to be auto-generated - maybe we should fix that...).